_posts/2019-11-07-open-source-camp-on-foreman-2019.md

```diff
@@ -0,0 +1,49 @@
+
+<img src="https://www.netways.de/wp-content/uploads/2019/11/Beitrag-2-300x225.jpg" style="max-width:100%;display: inline;float: left;">
+
+Next up was Bernhard Suttner, who wanted to give us a taste of Salted Foreman. Initially he explained what all that salt was about. The [SaltStack](https://www.saltstack.com/ "Salt Stack") a open source project written in python, can be used as a configuration management tool for Foreman. Salt excels at orchestrating cloud environments and network use-cases, but then we got to the Foreman relation. Running a 
+salt and Foreman environment means running a environment of managed hosts, which are salt minions and a *foreman_smart_proxy*, which will also be the salt master. He showed us what [salt in Foreman](https://github.com/theforeman/foreman_salt "foreman_salt") looks like and gave us some insight on how it works, but even more important from now on there are people dedicated to the project and some day the plugin might be as good as the puppet or ansible plugin. Salt is great and especially effective in terms of scalability. It's pretty straightforward to use and the initial setup is not so hard. We are excited for what is to come.
+
+<img src="https://www.netways.de/wp-content/uploads/2019/11/Beitrag-3-300x225.jpg" style="max-width:100%;display: inline;float: right;">
+
+*Provisioning on Azure Cloud* through Foreman by [Aditi Puntambekar](https://twitter.com/aditip16 "Aditi Puntambekar Twitter") was going to follow that one. Aditi made sure everyone is familiar with the extend of Foremans capabilities in terms of provisioning. This was especially important because Foremans capabilities differ from its usual when it comes to cloud provisioning. After a quick trip through the configuration of compute resources and imaged-based provisioning templates we went onward to the [Azure Resource Manager](https://azure.microsoft.com/en-us/features/resource-manager/ "Azure Resource Manager"). She explained how the Azure Resource Manager essentially worked, but what is interesting to us is the [foreman_azure_rm](https://github.com/theforeman/foreman_azure_rm "foreman_azure_rm"). Well and *foreman_azure_rm* does what you expect it to do. It adds the Microsoft Azure Resource Manager as a compute resource for the foreman. In her demo, she showed us how to use said resource and more.
```

Comment:
```suggestion
*Provisioning on Azure Cloud through Foreman* by [Aditi Puntambekar](https://twitter.com/aditip16 "Aditi Puntambekar Twitter") was going to follow that one. Aditi made sure everyone is familiar with the extend of Foremans capabilities in terms of provisioning. This was especially important because Foremans capabilities differ from its usual when it comes to cloud provisioning. After a quick trip through the configuration of compute resources and imaged-based provisioning templates we went onward to the [Azure Resource Manager](https://azure.microsoft.com/en-us/features/resource-manager/ "Azure Resource Manager"). She explained how the Azure Resource Manager essentially worked, but what is interesting to us is the [foreman_azure_rm](https://github.com/theforeman/foreman_azure_rm "foreman_azure_rm"). Well and *foreman_azure_rm* does what you expect it to do. It adds the Microsoft Azure Resource Manager as a compute resource for the foreman. In her demo, she showed us how to use said resource and more.
```